### PR TITLE
Add Sutch Coblized

### PR DESCRIPTION
**Pull request by charabaruk**
This adds my Sutch Coblized mod, and the ImpeREAL Empire Unique Cities - Sutch mod that it depends on. Also includes some housekeeping for Cobl that will be shared with my next PR, for Frostcrag Spire Cobl Alchemy Station.

### CODE REVIEW

**Review by sibir-ine — CHANGES_REQUESTED** [review]
This should be split into three PRs: one for the Cobl changes, one for *ImpeREAL Empire Unique Cities - Sutch*, & one for *Sutch Coblized*.

**Comment by charabaruk** [comment]
@sibir-ine I can easily those PRs, but seeing as they're each dependent on each other, this one would still have the commits of the other two. (Of course that would be moot if you do a fast-forward "merge" anyway). Give me a minute or two and I'll push up my cobl branch, create one for Sutch, and PR them both.

**Comment by charabaruk** [comment]
Now includes #1623 and #1624

File: masterlist.yaml [hunk 1]
```diff
@@ -34013,3 +34020,28 @@ plugins:
...
+        crc: 0xEB59AB31
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 2
+    clean:
+      - crc: 0x0872D0D1
+        util: 'TES4Edit v4.0.4'
+
+  - name: 'Sutch Coblized.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/52617'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - link: 'https://www.nexusmods.com/oblivion/mods/52617/'
```

URLs should have trailing slashes if possible for consistency & convenience (it makes it easier to search for Nexus IDs).

File: masterlist.yaml [hunk 2]
```diff
@@ -34013,3 +34020,28 @@ plugins:
...
+        name: 'Sutch Coblized'
+    req:
+      - *Cobl
+      - name: 'ImpeREAL Empire Unique Cities - Sutch.esp'
+        display: '[ImpeREAL Empire Unique Cities - Sutch](https://www.nexusmods.com/oblivion/mods/37867/)'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
```

The plugin already has them as masters, so these are redundant.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by charabaruk** [comment]
This should be good once #1624 is merged

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
Looks like GH still needs this PR rebased before it can be merged.

**Pull request merged**